src/server/PostManager.js

```diff
@@ -1,6 +1,34 @@
+import db from '../server/database';
+import {KeyUtils, ENTITY} from '../server/key-utils';
 
 export default class PostManager {
```

Comment:
I added 'entity' field in post object in my last commit.
lets move that field if we need it in other case.